src/probnum/filtsmooth/bayesfiltsmooth.py

```diff
@@ -10,9 +10,9 @@ class BayesFiltSmooth(ABC):
     Bayesian filtering and smoothing.
     """
 
-    def __init__(self, dynamod, measmod, initrv):
-        self.dynamod = dynamod
-        self.measmod = measmod
+    def __init__(self, dynamic_model, measurement_model, initrv):
+        self.dynamic_model = dynamic_model
+        self.measurement_model = measurement_model
```

Comment:
these used to be properties, and stored internally as dynamod. I found this unnecessary, (i) having a property that just aliases a variable and (ii), using a property for this in general. This pattern also applies to the driftmatrix stuff in sde.